test/mailers/password_mailer_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+
+class PasswordMailerTest < ActionMailer::TestCase
+  setup do
+    @person = people(:keith_registered)
+    @user = users(:keith)
+    credentials(:keith_password)
+
+    @settings = {
+      email_from: "teampeople@example.com",
+      product_name: "Product Name"
+    }
+    @features = {
+      password_reset_email: true,
+      email_sender_postmark: true
+    }
+  end
+
+  test "reset" do
+    os = "Windows"
+    browser = "Chrome"
+
+    stub_settings_and_features do
```

Comment:
this didn't actually do what I wanted it to do and set the feature in time to create the password reset routes. I added it to all the tests and then realized that CI was still broken. So I'll go back and remove it, but I think maybe as more env vars are added, we'll want a better way to set up the test environment so that running tests locally and running in CI behaves the same way regardless of what env vars we have set locally

Reply:
Hmm, that's an interesting thought. I'm pretty sure it's worked every other time, but I totally get that routes may be loaded early enough that it doesn't work for routes. Asking chatgpt it suggests that we simply do:

`Rails.application.reload_routes!`

I didn't test that but maybe all we need to do is add that to stub_settings and stub_features?